docs/intro.rst

```diff
@@ -40,15 +40,38 @@ respectively.
 
 Startup
 ^^^^^^^
+Then you can build all the services::
 
-Then you can start all the services::
+   $ docker-compose -f docker/docker-compose.dev.yml build
 
-   $ docker-compose -f docker/docker-compose.dev.yml up -d --build
+You need MusicBrainz database containing all the MusicBrainz music metadata for setting up your application. Data dumps provided from https://musicbrainz.org can be downloaded from https://musicbrainz.org/doc/MusicBrainz_Database/Download. ``mbdump.tar.bz2`` is the core MusicBrainz database including the tables for artist, release groups etc. ``mbdump-derived.tar.bz2`` contains annotations, user tags and search indexes. The core and the derived database covers almost all the data required for a CritiqueBrainz server.
+
+Then you can create and populate the database::
+
+   $ docker-compose -f docker/docker-compose.dev.yml run -v $DUMPS_DIR:/home/musicbrainz/musicbrainz-server -v $PWD/data/mbdata:/var/lib/postgresql/data/pgdata musicbrainz_db
+
+**Note** ``DUMP_DIR`` should be set to the path containing the downloaded dumps.
+
+You can skip downloading the dumps separately. Get the dumps, create and populate the database using::
+
+   $ docker-compose -f docker/docker-compose.dev.yml run -v $PWD/data/mbdata:/var/lib/postgresql/data/pgdata musicbrainz_db
+
+**Note** The above command downloads only the latest ``mbdump.tar.bz2`` and ``mbdump-derived.tar.bz2`` dumps for populating the database.
```

Comment:
I don't think this is needed, unless you wanted to draw attention to the fact that it only downloads these two files. You could add this to the paragraph above:
"You can automatically download ``mbdump.tar.bz2`` and ``mbdump-derived.tar.bz2`` and import them by not specifying a volume::"